.github/workflows/build.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           java-version: 11
 
       - name: Publish Artifacts
-        run: ./gradlew publishMavenPublicationToMavenCentralRepository paparazzi-gradle-plugin:publishPluginMavenPublicationToMavenCentralRepository
+        run: ./gradlew publishAllPublicationsToMavenCentralRepository
```

Comment:
1) layoutlib libraries need to be pushed prior to this under native libraries
2) the plugin marker isn't included as part of this.

Reply:
Ah ok, the issue is i forgot to update https://github.com/cashapp/paparazzi/blob/master/.github/workflows/build.yml#L67 per the fix in #343.

Reply:
Ahh, my expectation was that the root task would publish all publications for all projects.  Not sure what you mean by the plugin marker not being included in this.  It's just a maven publication, so should be included in all publications, I'd expect.  Is that not so?

Reply:
Doing a dry run shows that the layoutlib libraries and plugin marker are all included in the root task, unless I'm misunderstanding something.

```
rharter@XXHT23N9D9 paparazzi % ./gradlew --dry-run publishAllPublicationsToMavenCentralRepository
Type-safe project accessors is an incubating feature.

> Configure project :libs:layoutlib
No compatible plugin found in project layoutlib for publishing

> Configure project :libs:native-linux
No compatible plugin found in project native-linux for publishing

> Configure project :libs:native-macarm
No compatible plugin found in project native-macarm for publishing

> Configure project :libs:native-macosx
No compatible plugin found in project native-macosx for publishing

> Configure project :libs:native-win
No compatible plugin found in project native-win for publishing
:paparazzi-agent:compileKotlin SKIPPED
:paparazzi-agent:compileJava SKIPPED
:paparazzi-agent:processResources SKIPPED
:paparazzi-agent:classes SKIPPED
:paparazzi-agent:inspectClassesForKotlinIC SKIPPED
:paparazzi-agent:jar SKIPPED
:paparazzi:dokkaHtml SKIPPED
:paparazzi:dokkaJavadocJar SKIPPED
:paparazzi:kaptGenerateStubsKotlin SKIPPED
:paparazzi:kaptKotlin SKIPPED
:paparazzi:compileKotlin SKIPPED
:paparazzi:compileJava SKIPPED
:paparazzi:processResources SKIPPED
:paparazzi:classes SKIPPED
:paparazzi:inspectClassesForKotlinIC SKIPPED
:paparazzi:jar SKIPPED
:paparazzi:generateMetadataFileForMavenPublication SKIPPED
:paparazzi:generatePomFileForMavenPublication SKIPPED
:paparazzi:javaSourcesJar SKIPPED
:paparazzi:publishMavenPublicationToMavenCentralRepository SKIPPED
:paparazzi:publishAllPublicationsToMavenCentralRepository SKIPPED
:paparazzi-agent:generateMetadataFileForMavenPublication SKIPPED
:paparazzi-agent:generatePomFileForMavenPublication SKIPPED
:paparazzi-agent:javaSourcesJar SKIPPED
:paparazzi-agent:javadoc SKIPPED
:paparazzi-agent:simpleJavadocJar SKIPPED
:paparazzi-agent:publishMavenPublicationToMavenCentralRepository SKIPPED
:paparazzi-agent:publishAllPublicationsToMavenCentralRepository SKIPPED
:paparazzi-gradle-plugin:generatePomFileForPaparazziPluginMarkerMavenPublication SKIPPED
:paparazzi-gradle-plugin:publishPaparazziPluginMarkerMavenPublicationToMavenCentralRepository SKIPPED
:paparazzi-gradle-plugin:pluginVersion SKIPPED
:paparazzi-gradle-plugin:compileKotlin SKIPPED
:paparazzi-gradle-plugin:compileJava SKIPPED
:paparazzi-gradle-plugin:pluginDescriptors SKIPPED
:paparazzi-gradle-plugin:processResources SKIPPED
:paparazzi-gradle-plugin:classes SKIPPED
:paparazzi-gradle-plugin:inspectClassesForKotlinIC SKIPPED
:paparazzi-gradle-plugin:jar SKIPPED
:paparazzi-gradle-plugin:generateMetadataFileForPluginMavenPublication SKIPPED
:paparazzi-gradle-plugin:generatePomFileForPluginMavenPublication SKIPPED
:paparazzi-gradle-plugin:javaSourcesJar SKIPPED
:paparazzi-gradle-plugin:javadoc SKIPPED
:paparazzi-gradle-plugin:simpleJavadocJar SKIPPED
:paparazzi-gradle-plugin:publishPluginMavenPublicationToMavenCentralRepository SKIPPED
:paparazzi-gradle-plugin:publishAllPublicationsToMavenCentralRepository SKIPPED
:libs:layoutlib:cloneLayoutlib SKIPPED
:libs:layoutlib:emptyJavadocJar SKIPPED
:libs:layoutlib:emptySourcesJar SKIPPED
:libs:layoutlib:generatePomFileForMavenNativeLibraryPublication SKIPPED
:libs:layoutlib:publishMavenNativeLibraryPublicationToMavenCentralRepository SKIPPED
:libs:layoutlib:publishAllPublicationsToMavenCentralRepository SKIPPED
:libs:native-linux:emptyJavadocJar SKIPPED
:libs:native-linux:emptySourcesJar SKIPPED
:libs:native-linux:generatePomFileForMavenNativeLibraryPublication SKIPPED
:libs:native-linux:linuxJar SKIPPED
:libs:native-linux:publishMavenNativeLibraryPublicationToMavenCentralRepository SKIPPED
:libs:native-linux:publishAllPublicationsToMavenCentralRepository SKIPPED
:libs:native-macarm:emptyJavadocJar SKIPPED
:libs:native-macarm:emptySourcesJar SKIPPED
:libs:native-macarm:generatePomFileForMavenNativeLibraryPublication SKIPPED
:libs:native-macarm:macArmJar SKIPPED
:libs:native-macarm:publishMavenNativeLibraryPublicationToMavenCentralRepository SKIPPED
:libs:native-macarm:publishAllPublicationsToMavenCentralRepository SKIPPED
:libs:native-macosx:emptyJavadocJar SKIPPED
:libs:native-macosx:emptySourcesJar SKIPPED
:libs:native-macosx:generatePomFileForMavenNativeLibraryPublication SKIPPED
:libs:native-macosx:macJar SKIPPED
:libs:native-macosx:publishMavenNativeLibraryPublicationToMavenCentralRepository SKIPPED
:libs:native-macosx:publishAllPublicationsToMavenCentralRepository SKIPPED
:libs:native-win:emptyJavadocJar SKIPPED
:libs:native-win:emptySourcesJar SKIPPED
:libs:native-win:generatePomFileForMavenNativeLibraryPublication SKIPPED
:libs:native-win:winJar SKIPPED
:libs:native-win:publishMavenNativeLibraryPublicationToMavenCentralRepository SKIPPED
:libs:native-win:publishAllPublicationsToMavenCentralRepository SKIPPED

BUILD SUCCESSFUL in 460ms
```

Reply:
Sorry, I was too brief in my explanation.

Yes, the root task would publish all publications for all projects.  However, paparazzi (MavenPublication) requires shadowed/relocated layoutlib binaries to be published first under the MavenNativeLibrary publication type. See https://github.com/cashapp/paparazzi/blob/master/libs/README.md.  This is what I was referring to in 1.

The ultimate issue with the code is that my resolution in #343 only updated the paparazzi releasing README and not the actual CI publishing code 🤦 .  This is what I was referring to 2.

Layoutlib binaries are not SNAPSHOT versioned (since they map 1:1 with Google's layoutlib releases), so I anticipate that if I were to merge this PR, the `allPublications` task will fail because non-SNAPSHOT layoutlib binaries with that version already exists.  Even with layoutlib binaries had SNAPSHOT versions, since they rarely changed, these would be hefty and thus wasteful pushes to MavenCentral.

Layoutlib to Paparazzi is similar to how Haha was for Leakcanary, i.e., a shadowed version Google binary dependency, until py rewrote that.

Hope that explanation makes more sense.